machineconfiguration/v1/0000_80_machineconfigpool.crd.yaml

```diff
@@ -220,26 +217,33 @@ spec:
             status:
               description: MachineConfigPoolStatus is the status for MachineConfigPool resource.
               type: object
-              required:
-                - configuration
-                - degradedMachineCount
-                - machineCount
-                - readyMachineCount
-                - unavailableMachineCount
-                - updatedMachineCount
```

Comment:
this is currently not required in the MCO: https://github.com/openshift/machine-config-operator/blob/40571adccaa3ef49a823abe8d7494bcdc48f2021/install/0000_80_machine-config-operator_01_machineconfigpool.crd.yaml#L284